doc/Overview/Configuration/ConfigurationOverview.md

```diff
@@ -1,141 +1,158 @@
 
-For more documentation on configuration, read the references listed at the bottom.
+This feature uses [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration) for any configuration related work. For more documentation on configuration, read the references listed at the bottom.
 
 ## AppSettings
 
-To use `appsettings.json` file packaged as `EmbeddedResource` in the application:  
+To use `appsettings.json` file packaged as **EmbeddedResource** in the application:  
 
 ```csharp
 private IHost Host { get; }
 
-public App()
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseConfiguration(configure: builder => 
-            builder.EmbeddedSource<App>( includeEnvironmentSettings: true ))
-        .Build();
-    // ........ //
-}
-```  
+    var appBuilder = this.CreateBuilder(e)
+    .Configure(
+        host => host
+            .UseConfiguration(configure: builder => 
+                builder.EmbeddedSource<App>( includeEnvironmentSettings: true ))
+    );
+
+    Host = appBuilder.Build();
+...
+```
 
-To use `appsettings.json` file packaged as `Content` in the application:   
+To use `appsettings.json` file packaged as **Content** in the application:   
 
 ```csharp
 private IHost Host { get; }
 
-public App()
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseConfiguration(configure: builder => 
-            builder.ContentSource<App>( includeEnvironmentSettings: true ))
-        .Build();
-    // ........ //
-}
+    var appBuilder = this.CreateBuilder(e)
+    .Configure(
+        host => host
+            .UseConfiguration(configure: builder => 
+                builder.ContentSource<App>( includeEnvironmentSettings: true ))
+    );
+
+    Host = appBuilder.Build();
+...
 ```
 
-By default both `EmbeddedSource` and `ContentSource` methods will also add settings files that are specific to the current environment, `appsettings.<hostenvironment>.json` (eg `appsettings.development.json`). This can be disabled by setting the `includeEnvironmentSettings` argument to `false` (default value is `true`).
+By default, both `EmbeddedSource` and `ContentSource` methods will also create settings files that are specific to the current environment, `appsettings.<hostenvironment>.json` (eg `appsettings.development.json`). This can be disabled by setting the `includeEnvironmentSettings` argument to `false` (default value is `true`).
 
```

Comment:
Show example here of setting includeEnvironmentSetttings to false (instead of including the default value in the previous examples)